src/WebJobs.Script.WebHost/Diagnostics/DiagnosticEventTableStorageRepository.cs

```diff
@@ -17,6 +17,7 @@ namespace Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics
     public class DiagnosticEventTableStorageRepository : IDiagnosticEventRepository, IDisposable
     {
         internal const string TableNamePrefix = "AzureFunctionsDiagnosticEvents";
+        private const string CurrentEventVersion = "2024-05-01";
```

Comment:
TODO:// avoid hardcoding this twice

Reply:
The Table query is able to compare these just fine and it's working as expected (I tested with dates before and after May 1st and it was dropping the correct tables); would you prefer this to be something like 1.0?

Reply:
I just tried out the int version instead in the console app and it's not deleting any table with the value less than the current; given the string one is working i would like to stick with that